pkg/operator/ceph/object/rgw_test.go

```diff
@@ -172,6 +172,56 @@ func simpleStore() *cephv1.CephObjectStore {
 	}
 }
 
+func TestCreateObjectStoreWithKeystoneAndS3(t *testing.T) {
```

Comment:
As a complement to this test, let's add an inverse test (or modify an existing test) to add a check that verifies keystone isn't enabled when not configured.

I want to make sure that we are doing due diligence to ensure there aren't any side keystone-/swift-related side effects on traditional CephObjectStoreDeployments.

From my read of the config flow, I think to do that best would be to have a bit of code that ensures that none of the mon config store commands that run include `swift` or `keystone` keywords.

Reply:
I added assertions to the existing test `TestCreateObjectStore` in e9eb6b47